BlueprintUI/Sources/Layout/EqualStack.swift

```diff
@@ -65,7 +65,21 @@ extension EqualStack {
         var spacing: CGFloat
 
         func measure(in constraint: SizeConstraint, items: [(traits: Void, content: Measurable)]) -> CGSize {
-            let itemSizes = items.map { $1.measure(in: constraint) }
+            let totalSpacing = (spacing * CGFloat(items.count - 1))
```

Comment:
Will this cause any issues if there's no elements, and the spacing is negative?

Reply:
(I guess the array of sizes will just be empty too, so probably not)

Reply:
Yeah, the `.max() ?? 0` below handles the zero-element case, but we could probably add an early bail to be more explicit and avoid the negative spacing weirdness

Reply:
Added an early return